scriptmodules/supplementary/skyscraper.sh

```diff
@@ -275,6 +275,12 @@ function _init_config_skyscraper() {
     # Create the cache folder and add the sample 'priorities.xml' file to it
     mkUserDir "$scraper_conf_dir/cache"
     cp -f "$md_inst/.pristine_cfgs/priorities.xml.example" "$scraper_conf_dir/cache"
+
+    # Deploy programmable completion script
+    mkUserDir "$home/.bash_completion.d"
+    cp -f "$md_inst/.pristine_cfgs/Skyscraper.bash" "$home/.bash_completion.d"
+    chown $user:$user "$home/.bash_completion.d/Skyscraper.bash"
+    chmod 644 "$home/.bash_completion.d/Skyscraper.bash"
```

Comment:
The file is already `0644`, there's no need for the `chmod` here.

Reply:
Right. Cleared the line.